### ISSUE TITLE
使用smoke训练kitti数据集，增加的类别数（默认三类，我是9类），预测结果中3d预测框的h ，w， l都是负数

### ISSUE THREAD

**Issue by weihongwei-zg** [PaddlePaddle/Paddle3D#75]
训练时候只改了kitti_det.py中的CLASS_MAP字段。

**Comment by weihongwei-zg** [comment]
前3类'car': 0, 'cyclist': 1, 'pedestrian':2 的3d  bbox都是正常的。但我加的类别中只有2d框和3d中心点是正常的，3d长宽高都是负数

**Comment by weihongwei-zg** [comment]
请问如何改成自己需要的多类

**Comment by nepeplwu** [comment]
@wobushihuair 你的配置文件贴下吧，另外kitti_det.py的修改代码也发下

**Comment by weihongwei-zg** [comment]
_base_: '../_base_/kitti_mono.yml'

batch_size: 8
iters: 40000

train_dataset:
  transforms:
    - type: LoadImage
      reader: pillow
      to_chw: False
    - type: Gt2SmokeTarget
      mode: train
      num_classes: 4
      input_size: [1280, 384]
    - type: Normalize
      mean: [0.485, 0.456, 0.406]
      std: [0.229, 0.224, 0.225]

val_dataset:
  transforms:
    - type: LoadImage
      reader: pillow
      to_chw: False
    - type: Gt2SmokeTarget
      mode: val
      num_classes: 4
     input_size: [1280, 384]
    - type: Normalize
      mean: [0.485, 0.456, 0.406]
      std: [0.229, 0.224, 0.225]

optimizer:
  type: Adam

lr_scheduler:
  type: MultiStepDecay
  milestones: [24000, 36000]
  learning_rate: 1.25e-4

model:
  type: SMOKE
  backbone:
    type: DLA34
    # This will automatically save to ~/.paddle3d/pretrained/dla34/dla34.pdparams
    # pretrained: "https://bj.bcebos.com/paddle3d/pretrained/dla34.pdparams"
    pretrained:
  head:
    type: SMOKEPredictor
    num_classes: 4
    reg_channels: [1, 2, 3, 2, 2]
    num_chanels: 256
    norm_type: "gn"
    in_channels: 64
  depth_ref: [28.01, 16.32]
  dim_ref: [[3.88, 1.63, 1.53], [1.78, 1.70, 0.58], [0.88, 1.73, 0.67]]
  max_detection: 50
  pred_2d: True

**Comment by nepeplwu** [comment]
@weihongwei-zg `dim_ref` 表示的是各个类别的维度统计均值，如果是9类的话，这里应该填写 9 * 3 的列表
